tests/tests_pytorch/strategies/test_colossalai.py

```diff
@@ -231,18 +231,26 @@ def test_multi_gpu_checkpointing(tmpdir):
         precision=16,
         strategy="colossalai",
         callbacks=[ck],
+        num_sanity_val_steps=0,
```

Comment:
This and the test below (note the xfail) are due to a breaking change in the new colossal version: You can't run test/val after fit due to the requirement of the warmup phase. 

`AssertionError: You should run a completed iteration as your warmup iter`

We can fix this or improve the error messaging in a follow up.